source/user-manual/user-administration/password-management.rst

```diff
@@ -12,24 +12,24 @@ Password management
 
 Learn how to use the Wazuh passwords tool to manage your passwords. This tool allows you to change the passwords of both the :doc:`Wazuh indexer </getting-started/components/wazuh-indexer>` users, also known as internal users, and the :doc:`Wazuh manager API </user-manual/api/index>`  users.
 
-Among the Wazuh indexer users, it is worth mentioning the following: 
+Among the Wazuh indexer users, it is worth mentioning the following:
 
-- *admin*: is the default administrator user. It's used to log in to the web interface and for communications between Filebeat and the Wazuh indexer. If you change the *admin* password, you must update it in Filebeat.
+- *admin*: is the default administrator user. It's used to log in to the web interface and for communications between Filebeat and the Wazuh indexer. If you change the *admin* password, you must update it in Filebeat and Wazuh manager.
```

Comment:
We should use `Wazuh server` instead of `Wazuh manager` to be consistent with the rest of the page.

Reply:
Done: bd06a5336